buildenv/jenkins/common/build.groovy

```diff
@@ -29,9 +29,11 @@ def get_source() {
         OPENJ9_REPO_OPTION = (OPENJ9_REPO != "") ? "-openj9-repo=${OPENJ9_REPO}" : ""
         OPENJ9_BRANCH_OPTION = (OPENJ9_BRANCH != "") ? "-openj9-branch=${OPENJ9_BRANCH}" : ""
         OPENJ9_SHA_OPTION = (OPENJ9_SHA != "") ? "-openj9-sha=${OPENJ9_SHA}" : ""
+        OPENJ9_REFERENCE = "-openj9-reference=${OPENJDK_REFERENCE_REPO}"
```

Comment:
Should this not be OPENJ9_REFERENCE?

Reply:
No, it's a single folder that holds all the repos together. Originally only had the ext repos, hence the name.